File: 2189.py
teste = 0

while True:
    n = int(input())
    if n == 0:
        break

    teste += 1
    winner = 0

    nums = list(map(int, input().split()))
    for i, num in enumerate(nums, start=1):
        if num == i:
            winner = num

    print(f"Teste {teste}")
    print(winner)
    print()

File: 2189.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split(/\s+/).map(Number);

let idx = 0;
let teste = 0;

while (true) {
    const n = input[idx++];
    if (n === 0) break;

    teste++;
    let winner = 0;

    for (let i = 1; i <= n; i++) {
        const num = input[idx++];
        if (num === i)
            winner = num;
    }

    console.log(`Teste ${teste}`);
    console.log(winner);
    console.log("");
}
